test/pod/README.md

```diff
@@ -21,6 +21,7 @@ The [test-pod-tmpl.yaml](./test-pod-tmpl.yaml) can be used to define the test-po
 - `OPERATOR_IMAGE` is the etcd-operator image used for testing
 - `TEST_S3_BUCKET` is the S3 bucket name used for testing
 - `TEST_AWS_SECRET` is the secret name containing the aws credentials/config files.
+- `TEST_ABS_SECRET` is the secret name containing the abs credentials
```

Comment:
Can you revert the changes related under `test/pod/`. These would come into testing PR later.